ui/static/ui/js/learning_resources.jsx

```diff
@@ -6,73 +6,156 @@ define('learning_resources', [
   var StatusBox = Utils.StatusBox;
   var Select2 = Utils.Select2;
 
-  var VocabularyOption = React.createClass({
+  var TermListItem = React.createClass({
     render: function () {
-      var options = _.map(this.props.terms, function (term) {
+      return (
+        <li className="applied-term list-group-item">
+          <strong>{this.props.label}: </strong> {this.props.terms}
+        </li>
+      );
+    }
+  });
+
+  var VocabSelect2 = React.createClass({
+
+    render: function () {
+
+      var options = _.map(this.props.vocabs, function(vocab){
+        return {
+          id: vocab.vocabulary.slug,
+          text: vocab.vocabulary.name,
+        };        
+      });
+      
+      return <div className="col-sm-6">
+          <Select2
+            className="form-control"
+            placeholder={"Select a vocabulary"}
+            options={options}
+            allowClear={false}
+            onChange={this.handleChange}
+            values={this.props.selectedVocabulary.slug}
+            multiple={false}
+            />
+        </div>;
+    },
+    handleChange: function(e) {
+      var selectedValue = _.pluck(
+        _.filter(e.target.options, function(option) {
+          return option.selected && option.value !== null;
+        }), 'value');
+      this.props.updateVocabulary(_.pluck(this.props.vocabs, 'vocabulary'), selectedValue[0]);
+    }
+  });
+
+  var TermSelect2 = React.createClass({
```

Comment:
Why 2 is append with TermSelect because i havent seen any other TermSelect in the same file. Should it be more clear like TermSelect or TermSelectForLR or LRTermSelect or any other